src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/dao/GoogleDAO.scala

```diff
@@ -12,7 +12,7 @@ abstract class GoogleDAO {
 
   def createProject(projectName: String, billingAccountId: String): Future[ActiveOperationRecord]
   def pollOperation(operation: ActiveOperationRecord): Future[ActiveOperationRecord]
-  def cleanupDeployment(projectName: String): Unit
+  def cleanupDeployment(projectName: String): Future[Unit]
```

Comment:
Is the code that calls this method OK with it's return type changing from `Unit` to `Future[Unit]`?  I expected to see changes at the calling code too.

Reply:
The calling code is [here](https://github.com/broadinstitute/gpalloc/blob/643d1d82634a0945e7c8c6aede2189325b14c8b8/src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/monitor/ProjectCreationMonitor.scala#L88). It needs to clean up the deployment, delete the project, and remove the project from the database, but the order isn't important and if the Google calls fail then... oh well. I think when I wrote the calling code I was assuming that all three calls _were_ futures (hence the comment that they can all happen async).